### PR TITLE
Add emoji voto sample app to e2e tests

### PR DESCRIPTION
**Pull request by Zanda256**
…gmail.com

Signed-off-by: Sekiranda Hamza <sekirandahamza@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Thanks, @Zanda256 👏 

Feedback provided on today's Meshery CI call.

**Review by AugustasV — APPROVED** [review]
Looks good, it should point to service-mesh-patterns repo

File: .github/install/deploy.yaml [hunk 1]
```diff
@@ -3,5 +3,4 @@ services:
   linkerd:
     type: LinkerdMesh
     namespace: linkerd-system
-    version:  #will be inserted dynamically
-
+    version:  #will be inserted dynamically
```
Comment (AugustasV) [inline .github/install/deploy.yaml:6]: add new line

File: .github/install/deployWithSampleApp.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        controlplaneNS: linkerd-system
+        namespace: default
+        linkerdVersion: 2.11.3
+        profile: default
+  emojivoto:
+    type: $(#use https://raw.githubusercontent.com/service-mesh-patterns/service-mesh-patterns/master/samples/emojivoto.yaml)
+    settings: 
+      namespace: default
+    dependsOn:
+      - linkerdinstallation
```
Comment (AugustasV) [inline .github/install/deployWithSampleApp.yaml:15]: add new line

File: .github/install/installLinkerd.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    type: LinkerdMesh
+    namespace:  $(#ref.vars.controlplaneNS)
+    version: $(#ref.vars.linkerdVersion)
+    settings:
+      version: $(#ref.vars.linkerdVersion)
+      profile: $(#ref.vars.profile)
+    traits:
+      automaticSidecarInjection:
+        namespaces:
+          - $(#ref.vars.namespace)
```
Comment (AugustasV) [inline .github/install/installLinkerd.yaml:18]: add new line

File: .github/workflows/e2etests.yaml [hunk 4]
```diff
@@ -124,3 +123,116 @@ jobs:
...
+          rm data.json
+      - name: Commit
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          file_pattern: docs
+          commit_user_name: l5io
+          commit_user_email: ci@layer5.io
+          commit_author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          commit_options: '--signoff'
+          commit_message: '[Docs] Test status of adapter'
```
Comment (AugustasV) [inline .github/workflows/e2etests.yaml:238]: new line

File: .github/workflows/e2etests.yaml [hunk 5]
```diff
@@ -125,3 +124,116 @@ jobs:
...
+            cat ./.github/install/deployWithSampleApp.yaml
+      - name: Uploading file
+        uses: actions/upload-artifact@v2
+        with:
+          name: patternfile
+          path: ./.github/install/deployWithSampleApp.yaml 
+
+  TestLinkerdForServicemeshAndSampleApp:
+    needs: SetPatternfileForServicemeshAndSampleApp
+    uses: meshery/meshery/.github/workflows/test_adapters.yaml@master
```
Comment (Revolyssup) [inline .github/workflows/e2etests.yaml:161]: @Zanda256 Will you change this to refer to the v2 test_adapters that was created?

**Comment by leecalcote** [comment]
Merge conflict...

**Pull request closed without merging**